src/common/AndroidUtils.ts

```diff
@@ -728,8 +736,14 @@ export class AndroidSDKUtils {
         }
 
         // then ensure that the process is also running for the selected emulator
-        const findResult = CommonUtils.executeCommand(findProcessCommand);
-        return findResult != null && findResult.trim().length > 0;
+        let foundProcess = false;
+        try {
+            const findResult = CommonUtils.executeCommand(findProcessCommand);
+            foundProcess = findResult != null && findResult.trim().length > 0;
+        } catch {
+            // ignore and continue
```

Comment:
What kind of errors are we expecting to find here? Seems dangerous to just drop all errors here, unless we expect so common and innocuous ones.

Reply:
So the issue here is how `grep` works (at least on Mac). It seems that if you run `grep` on a set of results and if it does find a match then it will print the match to `stdout` and exits with a zero exit code. But if it doesn't find a match then it won't print anything to `stdout` **and** will exit with a non-zero exit code. Since the exit code is non-zero `childProcess.execSync()` considers that an error (even though in our case it really is not an error).

I think that we should have a `catch-all` logic here anyway. If for any reason we're unable to find a matching process then we should assume that the process is not running and attempt at relaunching the emulator.

Reply:
Ah! The "grep with no results is an error" issue. I know it well. 🙂 One way around it is to process the results internally, without grep. E.g.:
```javascript
// Some pseudo-code JS.
const procs = CommonUtils.executeCommand('ps -ax');
const procRegex = new RegExp(`qemu-system-x86_64.*${emulatorName}`);
for (const proc of procs.split('\n')) {
    if (procRegex.exec(proc) !== null) {
        // Do stuff.
    }
}
```
Even if we stick with grep (I'm okay with that), or change it to be more like above but keep the `catch` logic as you suggest (which I'd also support), let's log the error in the catch block, at the debug level, just in case we need to come back to it for troubleshooting, in the logs.

Reply:
I like the idea of having `grep` do the parsing instead of writing our own code (less code = less bugs 🙂 ). I'll add the log statement.